vochain/app.go

```diff
@@ -33,7 +35,10 @@ import (
 const (
 	// recheckTxHeightInterval is the number of blocks after which the mempool is
 	// checked for transactions to be rechecked.
-	recheckTxHeightInterval = 12
+	recheckTxHeightInterval = 6 * 5 // 5 minutes
+	// transactionBlocksTTL is the number of blocks after which a transaction is
+	// removed from the mempool.
+	transactionBlocksTTL = 6 * 10 // 10 minutes
```

Comment:
Are these values related to the consesus block time? If so, I would like to suggest to calculate the values on runtime and depending on the block time related parameters.